<launch>
    <node name="map_server" pkg="map_server" type="map_server" args="$(find mobiman_simulation)/config/test.yaml" />
</launch>